engine/server/engine/centralized_logs/client_implementations/kurtosis_backend/kurtosis_backend_logs_database_client.go

```diff
@@ -192,7 +193,8 @@ func streamServiceLogLines(
 				return
 			}
 
-			logLine := logline.NewLogLine(logLineStr)
+			// TODO: Parse out the timestamp from docker container logs string instead of time.Now
```

Comment:
@tedim52 I personally think what you have now is better, meaning: what you are logging is the timestamp we received the log whereas the timestamp from docker is when docker "managed" the log entry. I think it makes sense that we keep those two separate and it's fine to leave the docker timestamp embedded in the log line. Longer term I think we should move to structured logging so that we have a json entry that contains potentially many timestamps (for each actor that is involved in the parsing of the log line). With structured logging the user can then decide which timestamp is considered the "truth" (is it kurtosis timestamp, docker timestamp, service timestamp, etc.).

Reply:
Gotcha, this makes sense. I'll remove the TODO then.